CHANGELOG.md

```diff
@@ -29,6 +29,8 @@
   Print directories bold.
 * [#130](https://github.com/kowainik/summoner/issues/130):
   Show git revision version under `--version` command.
+* [#63](https://github.com/kowainik/summoner/issues/63):
+  Add `show ghc`, `show licenses` and `show license <LICENSE_NAME>` commands.
```

Comment:
I know think that it's better to not have `licenses` as a separate command. It would be even better if we could have less commands. And support:

```shell
$ summon show ghc
$ summon show license
$ summon show license MIT
```

This gives us open design space for putting something under the following command:

```shell
$ summon show ghc 8.4.3
```

Reply:
Here should be `show license` instead of `show licenses`